app/controllers/service_controller.rb

```diff
@@ -18,8 +18,6 @@ def show_list
 
   def show
     super
-    @options = {}
-    @view, @pages = get_view(Vm, :parent => @record, :parent_method => :all_vms, :all_pages => true)
   end
```

Comment:
Should this method just be removed entirely?  A method with just a super is a no-op